src/mode/modeNormal.ts

```diff
@@ -7,6 +7,7 @@ import TextEditor from './../textEditor';
 
 export default class CommandMode extends Mode {
 	private keyHandler : { [key : string] : () => void; } = {};
+	private registers: string[] = [];

```

Comment:
@jpoon could you please propose a suggestion where to put the registers for copy and paste commands? Thanks!